files/build_templates/per_namespace/swss.service.j2

```diff
@@ -14,7 +14,7 @@ After=updategraph.service
 After=interfaces-config.service
 BindsTo=sonic.target
 After=sonic.target
-Before=ntp-config.service pmon.service
+Before=ntp-config.service
```

Comment:
Doesn't it need to have mention/dependency on bootstrap-asic.service?

otherwise, this service may spawn independent of bootstrap-asic.service.
Should avoid letting this service spawn, start, enable etc. when bootstrap-asic.service has to come and stop, disable and mask it until ASIC for this service is detected online/operational?

Reply:
Marking resolved as we are not taking this approach